cli/build.rs

```diff
@@ -290,6 +290,8 @@ fn main() {
     return;
   }
 
+  println!("cargo:rustc-link-arg=-rdynamic");
```

Comment:
TODO: generate LINK definitions (`.def`) on Windows

Reply:
Had added that but for some reason linking `napi_exports.def` in `cargo test` build fails because symbols are not exported, while normal builds (using `cargo build`) work fine. No idea why its like that...